config/Data_Schema.toml

```diff
@@ -0,0 +1,469 @@
+
+[rusicoutdated]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = "nan"
+Min_values = 1120.0
+Max_values = 93059.0
+Possible_Categorical_Values = ["nan"]
+
+[frozensic]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = "nan"
+Min_values = 1300.0
+Max_values = 96090.0
+Possible_Categorical_Values = ["nan"]
+
+[rusic]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = "nan"
+Min_values = 1300.0
+Max_values = 96090.0
+Possible_Categorical_Values = ["nan"]
+
+[frozenemployees]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = "nan"
+Min_values = 0.0
+Max_values = 272527.0
+Possible_Categorical_Values = ["nan"]
+
+[employees]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = "nan"
+Min_values = 0.0
+Max_values = 272528.0
+Possible_Categorical_Values = ["nan"]
+
+[frozenemployment]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = "nan"
+Min_values = 1.0
+Max_values = 272527.0
+Possible_Categorical_Values = ["nan"]
+
+[employment]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = "nan"
+Min_values = 1.0
+Max_values = 272528.0
+Possible_Categorical_Values = ["nan"]
+
+[frozenfteemployment]
+Description = "nan"
+Deduced_Data_Type = "float"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = "nan"
+Min_values = 0.0
+Max_values = 177699.0
+Possible_Categorical_Values = ["nan"]
+
+[fteemployment]
+Description = "nan"
+Deduced_Data_Type = "float"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = "nan"
+Min_values = 0.0
+Max_values = 177699.5
+Possible_Categorical_Values = ["nan"]
+
+[frozenturnover]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = "nan"
+Min_values = 0.0
+Max_values = 55277352.0
+Possible_Categorical_Values = ["nan"]
+
+[turnover]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = "nan"
+Min_values = 0.0
+Max_values = 55277352.0
+Possible_Categorical_Values = ["nan"]
+
+[enterprisereference]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = ">=1"
+Min_values = 1001603625.0
+Max_values = 9891309165.0
+Possible_categorical_Values = ["nan"]
+
+[wowenterprisereference]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = ">=1"
+Min_values = 1001603625.0
+Max_values = 9891309165.0
+Possible_categorical_Values = ["nan"]
+
+[cellnumber]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "int"
+Length = "nan"
+Min_values = "nan"
+Max_values = "nan"
+Possible_Categorical_Values = [0]
+
+[currency]
+Description = "nan"
+Deduced_Data_Type = "str"
```

Comment:
This looks like it should be categorical data

Reply:
Yep, exactly right. Amended this and all others now.